sys-devel/gnuconfig/files/gnuconfig-20230731-llvm-libc.patch

```diff
@@ -0,0 +1,44 @@
+From b0fbd0b9c6cc4453e37fd2253221473a34549437 Mon Sep 17 00:00:00 2001
+From: Alfred Persson Forsberg <cat@catcream.org>
+Date: Wed, 2 Aug 2023 02:37:38 +0200
+Subject: [PATCH] config.sub: add LLVM libc support
```

Comment:
Is that an upstream patch?

Reply:
Yeah, we really need to agree with the LLVM people on this and then send it upstream.

Reply:
No, but I asked upstream about this and they said -llvm is fine for now. I'll ask again and try to get this upstreamed asap

Reply:
Any update on this?

Reply:
Sent a message on #libc now, also mentioned that every autotools based build is currently broken due to "unknown system". I guess I'll wait for a code owner to tell if '-llvm' is fine for fullbuild mode.

Reply:
[gnuconfig mailing list](https://lists.gnu.org/archive/html/config-patches/2023-10/threads.html#00004) <- waiting for another response

Also merged https://github.com/llvm/llvm-project/pull/69402/commits/d2f92688ba39c7d07fe79a780bb1b29a8fad750c and got code owner to agree on `arch-vendor-kernel-llvm` as triplet.

I guess we just wait for gnuconfig.